src/Elmish.WPF/BindingVmHelpers.fs

```diff
@@ -611,24 +613,25 @@ type Update
           this.Recursive(currentModel |> ValueOption.map b.Get, getCurrentModel >> b.Get, b.Get newModel, b.Dispatch, b.Binding)
 
 
-type Get(nameChain: string) =
+type Get<'a>(nameChain: string) =
 
-  member _.Base (model: 'model, binding: BaseVmBinding<'model, 'msg>) =
+  member _.Base (model: 'model, binding: BaseVmBinding<'model, 'msg, 'a>): Result<'a, GetError> =
```

Comment:
Please add this type annotation in a previous commit.  And (nit), please [add a space before that new colon](https://docs.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting#formatting-return-type-annotations).

Reply:
This type annotation was unnecessary. Removed.